mlos_bench/mlos_bench/environments/composite_env.py

```diff
@@ -59,6 +62,7 @@ def __init__(self,
 
         _LOG.debug("Build composite environment '%s' START: %s", self, tunables)
         self._children: List[Environment] = []
+        self._contexts: List[Environment] = []
```

Comment:
```suggestion
        self._child_contexts: List[Environment] = []
```

Reply:
Slight rename for clarity.
Without that, it's not entirely clear on a read through that a member `context` in that list even has a `run()` method for instance.

Reply:
yep